script.js

```diff
@@ -0,0 +1,18 @@
+const hamburgerIcon = document.querySelector('.hamburger_icon');
+const navMenu = document.querySelector('.nav_menu2');
+const navBranding = document.querySelector('.nav_branding');
+const navContainer = document.querySelector('.mobile_nav_container');
+
+hamburgerIcon.addEventListener('click', () => {
+  hamburgerIcon.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  navBranding.classList.toggle('active');
+  navContainer.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav_link').forEach((n) => n.addEventListener('click', () => {
+  hamburgerIcon.classList.remove('active');
+  navMenu.classList.remove('active');
+  navBranding.classList.remove('active');
+  navContainer.classList.remove('active');
+}));
```

Comment:
- [ ] Nice work implementing the features :+1: :sunglasses: But please make sure that a correct part of the page is displayed when clicking (or taping) any of the mobile menu options as requested in the project requirement.

![Screenshot from 2023-04-10 16-58-25](https://user-images.githubusercontent.com/91142432/230952300-f229aded-1877-40a9-9c34-bc925ecb5699.png)